testing/scenario_app/ios/Scenarios/ScenariosTests/FlutterViewControllerTest.m

```diff
@@ -0,0 +1,51 @@
+//
```

Comment:
Use our headers rather than the auto-generated one

Reply:
Changed it, didn't tweak the template.